style.css

```diff
@@ -0,0 +1,32 @@
+// неправильный комментарий
+
+.hEad {
+
+}
+
+
+
+/*нет пробелов в комментарии, это ошибка*/
+
+.wrap {
+
+  position: ;
+
+  display: block;
+
+}
+
+.main {color: aliceblue}
+
+.link {
+    color: aliceblue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/test_test/pullRequest?prid=5392485)